etna/core/middleware.py

```diff
@@ -0,0 +1,24 @@
+from datetime import datetime, timedelta
+from urllib.parse import quote
+
+from django.conf import settings
+
+
+class CookieMiddleware:
```

Comment:
We can afford to be a little more descriptive with the naming here:

```suggestion
class SetDefaultCookiePreferenceCookieMiddleware:
```